ciao-cli/main.go

```diff
@@ -139,8 +141,11 @@ const (
 	ciaoPasswordEnv    = "CIAO_PASSWORD"
 	ciaoComputePortEnv = "CIAO_COMPUTEPORT"
 	ciaoTenantNameEnv  = "CIAO_TENANT_NAME"
+	ciaoCACertFileEnv  = "CIAO_CA_CERT_FILE"
```

Comment:
@sameo Done (and for command line option too.) I also described using this instead of installing into the system keystore.